### PR TITLE
HTTP/2 for Jetty container

### PR DESCRIPTION
**Pull request by senivam**

### CODE REVIEW

File: containers/jetty-http2/src/main/java/org/glassfish/jersey/jetty/JettyHttp2ContainerFactory.java [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+        /**
+         * Starting the server if required
+         */
+        if (start) {
+            try {
+                // Start the server.
+                server.start();
+            } catch (final Exception e) {
+                throw new ProcessingException(LocalizationMessages.ERROR_WHEN_CREATING_SERVER(), e);
```
Comment (jansupol) [inline containers/jetty-http2/src/main/java/org/glassfish/jersey/jetty/JettyHttp2ContainerFactory.java:191]: The localizationMessages file is missing?

**Comment by jansupol** [comment]
Please update bom.pom

File: containers/jetty-http2/src/main/java/org/glassfish/jersey/jetty/JettyHttp2ContainerFactory.java (hunk 1 above)
Comment (senivam) [reply containers/jetty-http2/src/main/java/org/glassfish/jersey/jetty/JettyHttp2ContainerFactory.java:191]: it's taken from the original Jetty container. Shall it be separated?

Comment (jansupol) [reply containers/jetty-http2/src/main/java/org/glassfish/jersey/jetty/JettyHttp2ContainerFactory.java:191]: The best looks extending JettyHttpContainerProvider by `public <T> T createContainer(final Class<T> type, final Application application, Object parentContext)`

Comment (senivam) [reply containers/jetty-http2/src/main/java/org/glassfish/jersey/jetty/JettyHttp2ContainerFactory.java:191]: thanks, will try

**Review by jansupol — CHANGES_REQUESTED** [review]

**Review by jansupol — APPROVED** [review]

**Review by jbescos — APPROVED** [review]

**Pull request merged**